libs/core/src/components/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,59 @@
+import { html } from 'lit'
+import { property } from 'lit/decorators.js'
+import { classMap } from 'lit/directives/class-map.js'
+
+import { GdsElement } from '../../gds-element'
+import { tokens } from '../../tokens.style'
+import { gdsCustomElement } from '../../utils/helpers/custom-element-scoping'
+import {
+  withLayoutChildProps,
+  withMarginProps,
+  withSizeXProps,
+} from '../../utils/mixins/declarative-layout-mixins'
+import { IconChevronLeft } from '../icon/icons/chevron-left.component'
+import BreadcrumbCSS from './breadcrumb.styles'
+
+@gdsCustomElement('gds-breadcrumb', {
+  dependsOn: [IconChevronLeft],
+})
+export class GdsBreadcrumb extends withLayoutChildProps(
+  withSizeXProps(withMarginProps(GdsElement)),
+) {
+  static styles = [tokens, BreadcrumbCSS]
+
+  @property({ type: String })
+  size: 'large' | 'small' = 'large'
+
+  render() {
+    const elements = Array.from(this.children)
+    const secondToLastIndex = elements.length - 2
+
+    return html`
+      <nav
+        role="navigation"
+        aria-label="Breadcrumb"
```

Comment:
I think it would be good to be able to control the label here.

A default value of "Breadcrumbs" does make sense, but it needs to be localized then. Check here for instructions: https://lit.dev/docs/localization/overview/#extracting-messages

Reply:
I added a new label prop so it's up to users to change that, default will be 'breadcrumb'